src/containers/RegisterPage/registerForm.js

```diff
@@ -0,0 +1,127 @@
+        <div className="form-row">
+          <div className="col">
+            <label htmlFor="firstName">First name:</label>
+            <input
+              type="text"
+              className="form-control"
+              id="firstName"
+              onChange={formik.handleChange}
+              value={formik.values.firstName}
+            ></input>
+            {formik.errors.firstName ? (
+              <small className="form-text text-muted">
+                {formik.errors.firstName}
+              </small>
+            ) : null}
+          </div>
+          <div className="col">
+            <label htmlFor="lastName">Last Name:</label>
+            <input
+              type="text"
+              className="form-control"
+              id="lastName"
+              onChange={formik.handleChange}
+              value={formik.values.lastName}
+            ></input>
+            {formik.errors.lastName ? (
+              <small className="form-text text-muted">
+                {formik.errors.lastName}
+              </small>
+            ) : null}
+          </div>
+        </div>
+        <div className="form-group">
+          <label htmlFor="username">Username:</label>
+          <input
+            type="text"
+            className="form-control"
+            id="username"
+            onChange={formik.handleChange}
+            value={formik.values.username}
+          ></input>
+          {formik.errors.username ? (
+            <small className="form-text text-muted">
+              {formik.errors.username}
+            </small>
+          ) : null}
+        </div>
+        <div className="form-group">
+          <label htmlFor="email">Email:</label>
+          <input
+            type="email"
+            className="form-control"
+            id="email"
+            onChange={formik.handleChange}
+            value={formik.values.email}
+          ></input>
+          {formik.errors.email ? (
+            <small className="form-text text-muted">
+              {formik.errors.email}
+            </small>
+          ) : null}
+        </div>
+        <div className="form-group">
+          <label htmlFor="password">Password:</label>
+          <input
+            type="password"
+            className="form-control"
+            id="password"
+            onChange={formik.handleChange}
+            value={formik.values.password}
+          ></input>
+          {formik.errors.password ? (
+            <small className="form-text text-muted">
+              {formik.errors.password}
+            </small>
+          ) : null}
+        </div>
+        <div className="form-group">
+          <label htmlFor="password2">Confirm Password:</label>
+          <input
+            type="password"
+            className="form-control"
+            id="confirmPassword"
+            onChange={formik.handleChange}
+            value={formik.values.confirmPassword}
+          ></input>
+          {formik.errors.confirmPassword ? (
```

Comment:
Use `renderFormikError` helper here on multiple places (which I mentioned in the one of the comments)